pipelines/broad/dna_seq/germline/single_sample/exome/ExomeGermlineSingleSample.changelog.md

```diff
@@ -1,3 +1,9 @@
+# 2.5.0
+2021-08-05 
+
+* Added an optional step to reblock gVCFs, this step is included by default
```

Comment:
When we run this in production and want to reblock by default, is the pipeline now using the v1.1 calling intervals for  both haplotype caller and reblocking?

Reply:
HaplotypeCaller is on v1, so GVCFs don't change and people can go digging for calls in the padding.  Joint calling moves to v1.1 so the callset deliverable is smaller.

Reply:
Got it thank you!